### ISSUE TITLE
Redesign "SAN Adapters" UI

### ISSUE THREAD

**Issue by alinefm** [kimchi-project/ginger#318]
There are some issues in the SAN Adapters section:

![772ba3122505e661](https://cloud.githubusercontent.com/assets/4921718/14547574/b4c9310e-0285-11e6-8f1b-8c840a96c93f.jpg)

1. Why multiple selection is enabled if no action can be taken?

2. Paging should be shown only when needed

3. Refresh button and Filter input box are not aligned.

4. When no results are found, I don't see reason to display the table. Only an information about there are no SAN Adapters in the system

Said that, I suggest to change this section UI to something like below:

```
- SAN Adapters

| Refresh |                    | Filter                   |

| Name     | WWPN | WWNN | State   | In-Use/Max Ports | Speed   | Symbolic Name  |
| adapter1 | XXX  | YYY  | <state> | <ports>          | <speed> | <symbolicname> |
| adapter2 | XXX  | YYY  | <state> | <ports>          | <speed> | <symbolicname> |
```

All the widgets used for this mockup MUST follow Wok style.

When there is no adapter to display, only do:
```
- SAN Adapters
No results found in the system.
```

**Comment by samhenri** [comment]
@alinefm should we keep jQuery Bootgrid or use default wok.datagrid? Do we have confirmation that multi-selection is not necessary?

**Comment by danielhb** [comment]
Solved by c5f31932c67cfc6c8eb91f8ae824acdc0606f404
